src/packages/frontend/frame-editors/crm-editor/views/view-menu/search.tsx

```diff
@@ -151,12 +151,15 @@ function SearchBy({
 
 function SelectOperator({ fieldSpec, operator, onChange }) {
   const options = useMemo(() => {
-    if (fieldSpec.type == "boolean") {
+    if (fieldSpec.type === "boolean") {
       return [
         { value: "IS" as Operator, label: "IS" },
         { value: "IS NOT" as Operator, label: "IS NOT" },
       ];
     }
+    if (fieldSpec.type === "array") {
+      return [{ value: "ANY" as Operator, label: "ANY" }];
+    }
     return OPERATORS.filter((op) => op != "==").map((op: Operator) => {
```

Comment:
oh I see, here it appears again. So, maybe what's left is to filter "ANY" like "==". … but that shouldn't be a real problem...